### PR TITLE
added fixed role

### PR DESCRIPTION
**Pull request by AlexFry123**

### CODE REVIEW

File: src/app/constants/index.js [hunk 1]
```diff
@@ -2,6 +2,23 @@ import { firebaseConfig } from './firebaseConfig.js'
...
 import { route } from './route.js'
 import { color } from './color.js'
-import { role } from './role.js'
+import ROLE, { ROLE_KEYS, ROLE_VALUES } from './role.js'
 import { priority } from './priority.js'
-export { firebaseConfig, currency, route, color, role, priority }
+<<<<<<< HEAD
+=======
+
+>>>>>>> add98a8cf972cabb3f279e7ec17250ce38d90d3f
```
Comment (alexpassss) [inline src/app/constants/index.js:10]: merge conflict

File: src/app/domains/Role/components/views/RoleSimpleView/RoleSimpleView.template.js [hunk 2]
```diff
@@ -1,8 +1,18 @@
+import { Typography } from '@material-ui/core'
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+
 const RoleSimpleView = (props) => {
-  return <div>RoleSimpleView</div>
+  const { role } = props
+
+  const [currentRole, setCurrentRole] = useState(role)
```
Comment (alexpassss) [inline src/app/domains/Role/components/views/RoleSimpleView/RoleSimpleView.template.js:8]: fix everything from other pull requests first

**Review by alexpassss — CHANGES_REQUESTED** [review]

File: src/app/domains/Priority/components/select/PrioritySingleSelect/Select.style.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const styleSelect = (theme) => ({
+  root: {
+    minWidth: 100
+  }
+})
+export default styleSelect
```
Comment (GingerBoy14) [inline src/app/domains/Priority/components/select/PrioritySingleSelect/Select.style.js:6]: Why there is different style declaration from RoleSingleSelect.style.js file?

File: src/app/domains/Role/components/select/RoleSingleSelect/RoleSingleSelect.styles.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { makeStyles } from '@material-ui/core/styles'
+export const useStyles = makeStyles({
+  root: {
+    minWidth: 100,
+    textAlign: 'center'
+  }
+})
```
Comment (GingerBoy14) [inline src/app/domains/Role/components/select/RoleSingleSelect/RoleSingleSelect.styles.js:7]: Read previous comment

File: src/app/domains/Role/components/views/RoleSimpleView/RoleSimpleView.template.js [hunk 5]
```diff
@@ -1,8 +1,18 @@
...
+
+  return <Typography>{currentRole}</Typography>
 }
 
-RoleSimpleView.propTypes = {}
+RoleSimpleView.propTypes = {
+  role: PropTypes.string.isRequired
+}
 RoleSimpleView.defaultProps = {}
```
Comment (GingerBoy14) [inline src/app/domains/Role/components/views/RoleSimpleView/RoleSimpleView.template.js:16]: You can remove defaultProps

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

File: src/app/constants/role.js [hunk 6]
```diff
@@ -1,7 +1,3 @@
-const ROLE = { admin: 'admin', user: 'user', observer: 'observer' }
+const ROLES = ['admin', 'user', 'observer']
 
-const ROLE_KEYS = Object.keys(ROLE)
-const ROLE_VALUES = Object.values(ROLE)
-
-export default ROLE
-export { ROLE_KEYS, ROLE_VALUES }
+export { ROLES }
```
Comment (olenashevchuk) [inline src/app/constants/role.js:3]: export default should be here

**Review by GingerBoy14 — APPROVED** [review]

**Review by olenashevchuk — APPROVED** [review]

**Review by sashka2131 — APPROVED** [review]
ales good

**Pull request merged**